src/modules/builder/nav-bar/components/TemplatesSlider.tsx

```diff
@@ -48,9 +48,11 @@ export const TemplateSlider = () => {
           },
           '.splide__arrow--prev': {
             backgroundColor: 'transparent',
+            display: 'none !important', // #TODO: revert if more than templates available
```

Comment:
if no template are there further, is there any indicator? if present, we can hide arrow using that

Reply:
No, we don't have any indicator for no templates

Reply:
For now, planned to remove the hide logic, as there will be templates introduced in near future, then the issue won't even exist. and we also introduced cursor as disabled now so even if user is going to click the disabled arrow, they immediately get the feedback from ux perspective.